pkg/kratos/service.go

```diff
@@ -897,21 +955,50 @@ func (s *Service) HasNotEnoughLookupSecretsLeft(ctx context.Context, id string)
 
 func (s *Service) is1FAMethod(method string) bool {
 	switch method {
-	case "password", "webauthn", "oidc":
+	case "password", "oidc":
 		return true
+	case "webauthn":
+		return !s.oidcWebAuthnSequencingEnabled
 	default:
 		return false
 	}
 }
 
-func NewService(kratos KratosClientInterface, kratosAdmin KratosAdminClientInterface, hydra HydraClientInterface, authzClient AuthorizerInterface, tracer tracing.TracingInterface, monitor monitoring.MonitorInterface, logger logging.LoggerInterface) *Service {
+func (s *Service) hydrateKratosLoginFlow(ctx context.Context, flow *kClient.LoginFlow) error {
```

Comment:
this method is changing the flow via reference, wouldn't it be better to return a new flow with the enhanced/hydrated attributes?